munit-hedgehog/shared/src/main/scala/munit/HedgehogSuite.scala

```diff
@@ -0,0 +1,115 @@
+package munit
+
+import hedgehog._
+import hedgehog.core.{PropertyConfig, PropertyT, Seed}
+import hedgehog.predef.Monad
+
+import munit.internal.Compat._
+
+/**
+ * Provides the ability to write property based tests using the Hedgehog library.
+ *
+ * Properties are defined by calling one of the various `property` or `propertyF`
+ * methods and passing a `PropertyT[Result]` or `PropertyT[F[Result]]` respectively.
+ * For example:
+ * {{{
+ * property("additive identity") {
+ *   Gen.int(Range.linearFrom(0, Int.MinValue, Int.MaxValue)).forAll.map { n =>
+ *     assertEquals(n + 0, n)
+ *   }
+ * }
+ * }}}
+ *
+ * Note both Hedgehog `Result` and munit assertions are supported.
+ */
+trait HedgehogSuite extends FunSuite {
+
+  /**
+   * Provides the default configuration for running property tests.
+   * This can be overriden to change the default or can be changed on
+   * a test-by-test basis by using an overload of `property` and `propertyF`.
+   */
+  lazy val propertyConfig: PropertyConfig = PropertyConfig.default
+  lazy val propertySeed: Long = System.currentTimeMillis()
```

Comment:
Also, does it make sense to memoize the seed? IIUC in ScalaCheck the seed is generated fresh for each prop

Reply:
Minor, but would it make sense to use

```suggestion
  lazy val propertySeed: Seed = Seed.fromTime()
```

instead?

See https://github.com/hedgehogqa/scala-hedgehog/blob/master/core/shared/src/main/scala/hedgehog/core/Seed.scala#L48-L49

Reply:
Yeah I'd like to use `Seed` but need to figure out a way to convert an arbitrary seed to/from a string for reporting.

Reply:
I ended up using `String` myself in `ScalaCheckSuite`, since it's then easier for the user to set it when trying to reproduce a failure.

See the updated docs at https://scalameta.org/munit/docs/integrations/scalacheck.html#reproducing-a-property-failure